values.yaml

```diff
@@ -0,0 +1,30 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    repository: onetimesecret/onetimesecret
```

Comment:
Probably worth adding a registry option.

Example:

registry: "ghcr.io"  or "docker.io" as the default.

I think currently all the images are publishing to ghcr.io by default. We need to add docker.io to the main code repo but that shouldn't be a heavy lift.

Reply:
added registry in latest commit